training/distributed_training/pytorch/data_parallel/bert/pytorch_smdataparallel_bert_demo.ipynb

```diff
@@ -13,19 +13,19 @@
     "\n",
```

Comment:
<p>Would you add a description about the <code>train.sh</code> script? Why do you have the script as a workaround to run a training script? Does it include any other configs or settings? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Yes it is the entry point that launches the training script with options: https://github.com/aws/amazon-sagemaker-examples/blob/main/training/distributed_training/pytorch/data_parallel/bert/train.sh#L3</p>

Reply:
<p>Add a markdown cell below this cell:</p><p> </p><p>For <code>entry_point</code> of the estimator object, note that we specified the <code>train.sh</code> script, which includes a Python command to run the training script with various hyperparameters and configuration options. </p>